build.sh

```diff
@@ -12,19 +12,19 @@ LIBMIKMOD_COMMIT=738b1e8b11b470360b1b919680d1d88429d9d174
 LIBMIKMOD_DIR=/tmp/libmikmod
 
 # Clean, build, and install libdragon + tools
-make -j${CPU_COUNT} clobber
-make -j${CPU_COUNT} install tools-install
+make -j${CPU_COUNT} -I${N64_INST}/include clobber
+make -j${CPU_COUNT} -I${N64_INST}/include install tools-install
```

Comment:
maybe you can do `MAKE=make -j ${CPU_COUNT} -I${N64_INST}/include` and then use `${MAKE}` everywhere, rather than copying all the options in all instances.

Reply:
since we're at it, maybe I'd upgrade these scripts to [bash strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/)

Reply:
will do